web/admin.py

```
@@ -812,3 +818,131 @@ class PointsAdmin(admin.ModelAdmin):
+        ("Processing", {"fields": ("processed_by", "processed_at", "rejection_reason")}),
+        ("Timestamps", {"fields": ("applied_at",), "classes": ("collapse",)}),
+    )
+    actions = ["approve_selected", "reject_selected"]
+    def approve_selected(self, request, queryset):
+        for application in queryset.filter(status='pending'):
+            application.status = 'approved'
+            application.processed_by = request.user
+            application.save()
+        self.message_user(request, f"{queryset.filter(status='pending').count()} applications approved.")
+    approve_selected.short_description = "Approve selected applications"
+    def reject_selected(self, request, queryset):
```

Ruff (0.8.2)

Missing return type annotation for public function `reject_selected`

Add return type annotation: `None`

(ANN201)